src/sagemaker/utils.py

```diff
@@ -649,4 +651,67 @@ def _module_import_error(py_module, feature, extras):
     return error_msg.format(py_module, feature, extras)
 
 
+class DataConfig(abc.ABC):
+    """Abstract base class for creation for accessing data config hosted in AWS resourcs.
+
+    Provides a skeleton for customization by overriding of method fetch_data_config.
+    """
+
+    @abc.abstractmethod
+    def fetch_data_config(self):
+        """Abstract method implementing retrieval of data config from a pre-configured data source.
+
+        Returns:
+            object: The data configuration object for Example Notebooks
+        """
+
+
+class S3DataConfig(DataConfig):
+    """This class extends the DataConfig class to fetch a data config file hosted on S3"""
+
+    def __init__(
+        self,
+        sagemaker_session,
+        bucket_name="example-notebooks-data-config",
```

Comment:
may be you should pass this as an argument, it will make this code independent of example notebooks